src/middlewares/worker.js

```diff
@@ -1,7 +1,9 @@
 import createWorkerMiddleware from 'redux-worker-middleware';
 
-/* eslint-disable import/no-webpack-loader-syntax */
+/* eslint-disable */
```

Comment:
There were multiple errors associated with the proceeding line (associated with dependencies); it doesn't affect program flow and isn't something to worry about